LICENSE.rtf

```diff
@@ -0,0 +1,207 @@
+      agreed to in writing, Licensor provides the Work (and each\par
+      Contributor provides its Contributions) on an "AS IS" BASIS,\par
+      WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or\par
+      implied, including, without limitation, any warranties or conditions\par
+      of TITLE, NON-INFRINGEMENT, MERCHANTABILITY, or FITNESS FOR A\par
+      PARTICULAR PURPOSE. You are solely responsible for determining the\par
+      appropriateness of using or redistributing the Work and assume any\par
+      risks associated with Your exercise of permissions under this License.\par
+\par
+   8. Limitation of Liability. In no event and under no legal theory,\par
+      whether in tort (including negligence), contract, or otherwise,\par
+      unless required by applicable law (such as deliberate and grossly\par
+      negligent acts) or agreed to in writing, shall any Contributor be\par
+      liable to You for damages, including any direct, indirect, special,\par
+      incidental, or consequential damages of any character arising as a\par
+      result of this License or out of the use or inability to use the\par
+      Work (including but not limited to damages for loss of goodwill,\par
+      work stoppage, computer failure or malfunction, or any and all\par
+      other commercial damages or losses), even if such Contributor\par
+      has been advised of the possibility of such damages.\par
+\par
+   9. Accepting Warranty or Additional Liability. While redistributing\par
+      the Work or Derivative Works thereof, You may choose to offer,\par
+      and charge a fee for, acceptance of support, warranty, indemnity,\par
+      or other liability obligations and/or rights consistent with this\par
+      License. However, in accepting such obligations, You may act only\par
+      on Your own behalf and on Your sole responsibility, not on behalf\par
+      of any other Contributor, and only if You agree to indemnify,\par
+      defend, and hold each Contributor harmless for any liability\par
+      incurred by, or claims asserted against, such Contributor by reason\par
+      of your accepting any such warranty or additional liability.\par
+\par
+   END OF TERMS AND CONDITIONS\par
+\par
+   APPENDIX: How to apply the Apache License to your work.\par
+\par
+      To apply the Apache License to your work, attach the following\par
+      boilerplate notice, with the fields enclosed by brackets "[]"\par
+      replaced with your own identifying information. (Don't include\par
+      the brackets!)  The text should be enclosed in the appropriate\par
+      comment syntax for the file format. We also recommend that a\par
+      file or class name and description of purpose be included on the\par
+      same "printed page" as the copyright notice for easier\par
+      identification within third-party archives.\par
+\par
+   Copyright [yyyy] [name of copyright owner]\par

```

Comment:
This is identical with https://github.com/mackerelio/mackerel-agent/blob/master/LICENSE#L189
Please open another issue if needed?